report/src/main/java/com/buschmais/jqassistant/core/report/api/ReportHelper.java

```diff
@@ -143,6 +143,7 @@ private void log(LoggingStrategy loggingStrategy, ExecutableRule rule, List<Stri
         loggingStrategy.log(header);
         loggingStrategy.log(type + ": " + rule.getId());
         loggingStrategy.log("Severity: " + severityInfo);
+        loggingStrategy.log("Number of violations: " + resultRows.size());
```

Comment:
"violations" is misleading in the case of aggregated results (e.g. count(*)), would prefer "rows". Within the context of printing the result to the user the meaning should still be clear to him.

Reply:
In case of Concepts, you are right. I had the Constraints in mind, so I used "violations" - because only for Constraints are the result lines printed out.
Maybe also "result lines" or "result rows" could be used.

Reply:
It holds even in the case of constraints if the query returns just the count of detected violations.